code/modules/projectiles/projectile/bullets.dm

```diff
@@ -210,6 +210,29 @@
 	damage = 50
 	weaken = 1
 
+/obj/item/projectile/bullet/rifle/slugger
+	name = "slugger round"
+	damage = 80
+	stun = 3
+	weaken = 3
+	penetrating = 5
+	armor_penetration = 15
+	maiming = TRUE
+	maim_rate = 3
+	maim_type = DROPLIMB_BLUNT
+
+/obj/item/projectile/bullet/rifle/slugger/on_hit(var/atom/target, var/blocked = 0)
+	if(isobj(target))
+		var/obj/throwee = target
+		var/throwdir = get_dir(firer, target)
+		throwee.throw_at(get_edge_target_turf(target, throwdir), 3, 3)
+	else if(ismob(target))
+		var/mob/throwee = target
+		var/throwdir = get_dir(firer, target)
+		throwee.throw_at(get_edge_target_turf(target, throwdir), 3, 3)
```

Comment:
Since these two things are the same, you can just cast the target as /atom/movable/throwee and then throw it.

Reply:
well, i believe i tried that at first, then it tossed a bunch of runtimes at me in testing